WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -8,19 +8,53 @@ import Yosemite
 //
 final class DashboardViewController: UIViewController {
 
+    // MARK: Constants
+    enum Constants {
+        static let animationDuration = 0.2
+    }
```

Comment:
I think we already have a constants extension at the end of this file, could we move this down, please?

https://github.com/woocommerce/woocommerce-ios/blob/858a992cc766d41a0ab50a9dd9a872521b970ba6/WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift#L214-L221

Reply:
Sure, moved!